tests/common.go

```diff
@@ -3595,6 +3597,8 @@ func CreateCloudCredential(provider, credName string, uid, orgID string, ctx con
 				},
 			},
 		}
+	case drivers.ProviderNfs:
+		return nil
```

Comment:
Add a log message before return stating `Creation of NFS backuplocation does not require Cloud Credential`

Reply:
Done. Added a warning message saying cloud credential creation is not required for NFS provider.